docs/getting_started.md

```diff
@@ -1,36 +1,35 @@
-# TensorFlow Encrypted
+# Getting Started with tf-encrypted
```

Comment:
We can probably just remove this now that we have the main documentation. I don't think anything links here an maintaining two duplicate guides is a lot of work!

Reply:
not sure I follow -- which part should be removed?

Reply:
This file entirely as we have `docs/source/getting_started.rst` which is available at https://tf-encrypted.readthedocs.io/en/latest/usage/getting_started.html